spec/index.md

```diff
@@ -651,6 +669,46 @@ The functionality exposed via the [The IC management canister](#ic-management-ca
 
 :::
 
+### Request: Sync Call {#http-sync-call}
+
+In order to make a synchronous update call to a canister, the user makes a POST request to `/api/v2/canister/<effective_canister_id>/sync_call`. The request body consists of an authentication envelope with a `content` map with the following fields:
+
+-   `request_type` (`text`): Always `sync-call`
+
+-   `sender`, `nonce`, `ingress_expiry`: See [Authentication](#authentication)
+
+-   `canister_id` (`blob`): The principal of the canister to call.
+
+-   `method_name` (`text`): Name of the canister method to call
+
+-   `arg` (`blob`): Argument to pass to the canister method
+
+The HTTP response to this request can have the following responses:
+
+-   202 HTTP status with empty body. Implying the request was accepted by the IC for further processing. Users should use [`read_state`](#http-read-state) to determine the status of the call.
+
+-   200 HTTP status with a non-empty body. The response contains the canister response, which is either a `reply` or a `reject`.
+    
+    -   If the update call resulted in a reply, the response is a CBOR (see [CBOR](#cbor)) map with the following fields:
+
+        -   `reply` (`blob`):  A certificate (see [Certification](#certification)) with the subtree at `/request_status/<request_id>` and `/time`.
+
+    -  If the update call resulted in a reject, the response is a CBOR (see [CBOR](#cbor)) map with the following fields:
```

Comment:
We should not return two different types of CBOR maps without giving the user a way to tell what to try parsing. Please see my suggestion on the PR: let's introduce an enumeration and wrap the two CBOR maps (for reply and reject) as variants of that enumeration.

Reply:
I added a `status` field to both CBOR maps with the value set to either `rejected` or `replied`. This is the same way we handle reject and reply for Query calls.  https://internetcomputer.org/docs/current/references/ic-interface-spec#http-query

Is this fine by you?

Reply:
Yes. That makes sense - Thank you for taking inspiration from query calls!